internal/driver/glfw/testdata/menu_bar_kbdctrl_traverse_menu_down_2.xml

```diff
@@ -16,7 +16,7 @@
 				<rectangle size="0x0"/>
 				<text bold pos="12,8" size="53x21">Button</text>
 			</widget>
-			<widget size="300x29" type="*glfw.MenuBar">
+			<widget backgroundColor="button" size="300x29" type="*glfw.MenuBar">
```

Comment:
this shouldn't be here - backgroundColor was removed

Reply:
A test run was missed somehow, should be fixed now